cvat-cli/src/cvat_cli/_internal/common.py

```diff
@@ -27,20 +34,40 @@ class CriticalError(Exception):
     pass
 
 
-def get_auth_factory(s: str) -> Callable[[str], tuple[str, str]]:
+CVAT_ACCESS_TOKEN_ENV_VAR = "CVAT_ACCESS_TOKEN"  # nosec - a variable name declaration
```

Comment:
What do you think about moving support for this variable to the SDK? Seems like it would be useful for custom scripts if the `Client` class would automatically fetch the token from the environment.